vllm/entrypoints/openai/protocol.py

```diff
@@ -716,7 +716,7 @@ class BatchResponseData(OpenAIBaseModel):
     request_id: str
 
     # The body of the response.
-    body: Union[ChatCompletionResponse, ]
+    body: Optional[Union[ChatCompletionResponse, ]] = None
```

Comment:
```suggestion
    body: Optional[ChatCompletionResponse] = None
```

Reply:
Done. In theory the 'body' can also be `EmbeddingResponse` and `CompletionResponse`. We can add them later.